docs/clusterfuzzlite.md

```diff
@@ -38,6 +38,7 @@ ClusterFuzzLite is based on [ClusterFuzz].
 
 ## Supported CI Systems
 - [GitHub Actions]
+- [GitLab]
```

Comment:
This link doesn't go anywhere. Add target to line 79.

Reply:
indeed